apps/options-board/components/Board/Board.js

```diff
@@ -47,6 +53,10 @@ class Board extends HTMLElement {
   }
 
   set data (obj) {
+    if (!obj) {
+      return
```

Comment:
@save91 attenzione: in questo modo l'attributo rimane quello vecchio se prima passi un valore valido e poi uno non valido. In questo caso forse conviene fare un `removeAttribute`

Reply:
qui si manifesta un caso strano, la funzione viene richiamata prima del dovuto e settando un "undefined" sul valore di data. Per questo ho aggiunto un controllo qui, non sono riuscito a capire dove venga richiamato

Reply:
mi serviva per gestire un corner case ma ora non si manifesta più. Lo tolgo